taqueria-plugin-ligo/compile.ts

```diff
@@ -15,9 +15,12 @@ const getInputFilename = (opts: Opts) => (sourceFile: string) => {
 }
 
 const getCompileCommand = (opts: Opts) => (sourceFile: string) => {
-    const {projectDir} = opts
+    let {projectDir} = opts
```

Comment:
Hey @GImbrailo, I'd suggest one change:
```js
const projectDir = process.env.PROJECT_DIR ??  opts.projectDir

if (!projectDir) throw `No project directory provided`
```

Later, once my 620-operations branch is merged in, we can rewrite as:
```js
const projectDir = SanitizedAbsPath.create(process.env.PROJECT_DIR ?? opts.projectDir)
```

Reply:
Thanks @mweichert I implemented your suggestion and it is working as expected.